docs/.storybook/preview-head.html

```diff
@@ -1,11 +1,20 @@
-<link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i' rel='stylesheet' type='text/css'></link>
+<link rel="preconnect" href="https://fonts.googleapis.com"></link>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin></link>
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,400&display=swap" rel="stylesheet"></link>
 <style>
   html,
   body {
+    color: var(--font-color-for-text);
+    font-family: var(--font-family);
+    font-size: 13px;
     margin: 0;
     padding: 0;
-    font-size: 13px;
-    font-family: 'Open Sans', sans-serif;
+  }
+
+  @supports (font-variation-settings: normal) {
+    html, body {
+      font-family: var(--font-family-variable);
+    }
```

Comment:
This enables the [new variable fonts](https://fonts.google.com/knowledge/introducing_type/introducing_variable_fonts).